titan-web-client/package.json

```diff
@@ -3,78 +3,79 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fortawesome/fontawesome": "^1.1.8",
-    "@fortawesome/fontawesome-free-regular": "^5.0.13",
-    "@fortawesome/fontawesome-free-solid": "^5.0.13",
-    "@fortawesome/react-fontawesome": "^0.1.3",
-    "autoprefixer": "^9.3.1",
-    "axios": "^0.18.0",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^10.0.1",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^8.0.4",
+    "@fortawesome/fontawesome": "^1.1.3",
```

Comment:
The last PR merged into master bumped many dependencies by major versions, including webpack which broke the app when running `yarn start`. These dependencies should be incrementally updated on an as-needed basis.